app.js

```diff
@@ -14,12 +14,52 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index.hbs');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi =>
+      //console.log('Beers from the database: ', beersFromApi)
+      res.render('beers.hbs', {
+        beersFromApi
+      }))
+    .catch(error => console.log(error));
+
+})
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
+      //console.log(responseFromAPI)
+      res.render('random-beer.hbs', {
+        responseFromAPI
+      })
+    })
+    .catch(error => console.log(error));
+})
+
+app.get('/beers/beer-:id', (req, res) => {
+
+  let id = req.params.id
+
+  punkAPI
+    .getBeer(id)
+    .then(id =>
```

Comment:
in line 54 you are passing the beer id, however in like 55 you are getting a beer obj as a response, not another id. Be careful with variable naming :)